src/app/secure-layout/secure-layout.component.css

```diff
@@ -41,9 +41,10 @@
     display: flex;
     flex-direction: column;
     min-height: calc(100vh - 50px);
-    margin-top: 50px;
+    margin-top: 60px;
+    position: relative;
 }
-
+\
```

Comment:
:) 

p.s. When you make some changes and create a pull request, check the files you've changed, you will notice these things.